src/routes/routes.ts

```diff
@@ -27,4 +27,4 @@ router.get(urls.CONFIRM_COMPANY, confirmCompanyRoute.get);
 
 router.post(urls.CONFIRM_COMPANY, confirmCompanyRoute.post);
 
-router.get(urls.COMPANY_AUTH_PROTECTED_BASE, (req, res) => res.send(req.params));
+router.get(urls.TRADING_STATUS, (req, res) => res.send("Trading Status"));
```

Comment:
Is "trading-status" a placeholder for something else that gets added into the url the background of the auth code library?

Reply:
https://companieshouse.atlassian.net/browse/NCS-123
I'm just following the story.

Reply:
its the next page in the journey after confirm company